rocAL/rocAL/source/pipeline/master_graph.cpp

```diff
@@ -682,7 +682,7 @@ MasterGraph::copy_out_tensor(void *out_ptr, RocalTensorFormat format, float mult
         for( auto&& out_image: output_buffers)
         {
             unsigned int single_image_size = w * c * h;
-            #pragma omp parallel for
+            #pragma omp parallel for num_threads(n)
```

Comment:
Shouldn't this be set to IBS?

Reply:
I had removed that as we were planning to issue a seperate PR for the performance based changes. 
Adding it back